File: src/test/java/me/rasztabiga/fridgy/products/service/mapper/ProductUnitMapperTest.java
package me.rasztabiga.fridgy.products.service.mapper;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class ProductUnitMapperTest {

    private ProductUnitMapper productUnitMapper;

    @BeforeEach
    public void setUp() {
        productUnitMapper = new ProductUnitMapperImpl();
    }
}

File: src/test/kotlin/me/rasztabiga/fridgy/products/service/mapper/ProductUnitMapperTest.kt
package me.rasztabiga.fridgy.products.service.mapper

import org.junit.jupiter.api.BeforeEach

class ProductUnitMapperTest {

    private lateinit var productUnitMapper: ProductUnitMapper

    @BeforeEach
    fun setUp() {
        productUnitMapper = ProductUnitMapperImpl()
    }
}
